WalletWasabi.Fluent/Views/Wallets/Home/Tiles/CommonStyles.axaml

```diff
@@ -0,0 +1,23 @@
+<Styles xmlns="https://github.com/avaloniaui"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
+  <Design.PreviewWith>
+    <Border Padding="20">
+      <HeaderedContentControl Header="Header">
+        <TextBlock>Content</TextBlock>
+      </HeaderedContentControl>
+    </Border>
+  </Design.PreviewWith>
+
+  <Style Selector="HeaderedContentControl">
+    <Setter Property="Template">
+      <ControlTemplate>
+        <DockPanel>
+          <ContentPresenter DockPanel.Dock="Top" Content="{TemplateBinding Header}"
+                            Classes="h8 bold"
+                            ContentTemplate="{TemplateBinding HeaderTemplate}" />
+          <ContentPresenter Content="{TemplateBinding Content}" ContentTemplate="{TemplateBinding ContentTemplate}" />
+        </DockPanel>
+      </ControlTemplate>
+    </Setter>
+  </Style>
+</Styles>
```

Comment:
OK so... 

This file should be stored in the `Styles` folder, and the name should be `HeaderedContentControl`.
And the file should be included in the `Styles.axaml`

And to avoid applying it on every `HeaderedContentControl` you should use a style class for it like `HeaderedContentControl.tileHeader`.

And use that `tileHeader` style class in the styles.